Samples.Objc/Samples.ObjC/API/Endpoints/FetchAccessTokenEndpoint.m

```diff
@@ -1,37 +1,32 @@
 //
 //  FetchAccessTokenEndpoint.m
-//  PayPalCheckout-Samples-iOS-Objc
+//  PayPalNativeCheckoutObjC
 //
-//  Created by Haider Khan on 5/18/20.
-//  Copyright © 2020 PayPal. All rights reserved.
+//  Created by Nguyen, The Nhat Minh on 3/14/21.
 //
 
 #import "FetchAccessTokenEndpoint.h"
+#import "PayPalAPI.h"
 
 @implementation FetchAccessTokenEndpoint
 
 - (NSString *)path {
   return @"auth/token";
 }
 
-- (NSString *)method {
-  return @"POST";
-}
-
 - (NSURLRequest *)urlRequestFor:(FetchAccessTokenRequest *)request {
-  NSString *baseUrlString = [[PayPalAPI shared] nodeAppBaseURL];
-  NSString *fullPath = [NSString stringWithFormat:@"%@%@", baseUrlString, [self path]];
-  NSURL *url = [NSURL URLWithString:fullPath];
+  NSString *urlString = [NSString stringWithFormat:@"%@%@", [PayPalAPI.shared nodeAppBaseURL], [self path]];
+  NSURL *url = [NSURL URLWithString:urlString];
```

Comment:
I think we can move this inline now that we are relying on `PayPalAPI.shared` like so:
```  
NSURL *url = [NSURL URLWithString: [PayPalAPI.shared nodeAppBaseURL]];
```

Reply:
Here I think `NSURL *url = [NSURL URLWithString:[NSString stringWithFormat:@"%@%@", [PayPalAPI.shared nodeAppBaseURL], [self path]]]` maybe kinda long so I decided to leave them in two lines :(

Reply:
I don't think you need `stringWithFormat` at all. I tested with replacing both lines with `NSURL *url = [NSURL URLWithString: [PayPalAPI.shared nodeAppBaseURL]];` and it fetched the token without issue!